Angular/src/app/paradigm-list/paradigm-list.component.scss

```diff
@@ -0,0 +1,5 @@
+button {
+    &.button {
+        cursor: pointer;
```

Comment:
this is really helpful also, I like that the cursor now changes to show that it is a button